### DISCUSSION TITLE
TRMM entirely behind a SDWAN

### DISCUSSION THREAD

**Discussion by applebyp** [amidaware/tacticalrmm#1327] [Q&A] [answered]
Hi all. I've been using TRMM (sponsored :) ) for a while and would like to put a new install entirely behind a SDWAN such as ZeroTier. I've tried it but get the NGINX login page but no 2FA page and then after a timeout I get a backend offline. I have no UFW or ZeroTier firewall running at this stage.
I've set DNS to resolve rmm. api. mesh. to the same ZeroTier IP.
Is this just a case of one of the services listening only on the LAN IP? If so where can I change that to listen on all IPs?
Cheers.
P.

**Comment by dinger1986** [comment, marked as answer]
Where's the server hosted?

Services should be listening on all interfaces.

What's in your /etc/hosts file?

**Reply by applebyp** [reply]
Server is hosted in my office lab, test connections from another office lab.

**Reply by applebyp** [reply]
I'll report on /etc/hosts tomorrow when I'm back in.

**Comment by wh1te909** [comment]
your sponsorship is no longer active btw, you cancelled it on Sept 23

**Reply by applebyp** [reply]
Oh thanks I'll get that started again.

**Reply by applebyp** [reply]
🎉 You're now sponsoring @amidaware for $50 a month

**Comment by applebyp** [comment]
Not sure what happened but all's working now. Split DNS on the local LAN (I want TRMM to look after a couple of Windows DCs so can't use the SDWAN agent on there.)
